.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Feature Request 💡
+about: Suggest a new idea for the project.
+labels: "request-feature"
```

Comment:
Are we renaming the existing label, or creating a new one?

https://github.com/rapid7/metasploit-framework/labels?q=feature